Assets/Scripts/Missions/MissionOutcomes/MoneyOutcome.cs

```diff
@@ -5,11 +5,10 @@
 [CreateAssetMenu(fileName = "MoneyOutcome", menuName = "ScriptableObjects/Missions/Outcomes/MoneyOutcome", order = 1)]
 public class MoneyOutcome : MissionOutcome
 {
-    public float probability;
     public float moneyMin;
     public float moneyMax;
 
-    public override void Process()
+    public override void Process(Mission mission)

```

Comment:
notice how you dont use the params here. Make 2 method overloads, one with the mission param and one without